test/browser_tests/spec_suites/wagtail/wagtail-page-states-pages.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var DraftPage = require(
+  '../../page_objects/page_wagtail_states_pages.js' ).draftpage;
+var SharedPage = require(
+  '../../page_objects/page_wagtail_states_pages.js' ).sharedpage;
+var LivePage = require(
+  '../../page_objects/page_wagtail_states_pages.js' ).livepage;
+var LiveDraftPage = require(
+  '../../page_objects/page_wagtail_states_pages.js' ).livedraftpage;
+
+describe( 'Wagtail Draft Page', function() {
+  var page;
+
+  beforeAll( function() {
+    page = new DraftPage();
+    page.get();
+  } );
+
+  it( 'should not load in a browser',
+    function() {
+      expect( 404 );
```

Comment:
Yeah, also one thing that is useful/ we should be doing as we write tests is to make sure our tests fail with other input.